x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/es_deprecations.tsx

```diff
@@ -77,7 +77,7 @@ const getBatchReindexLink = (docLinks: DocLinksStart) => {
   return (
     <FormattedMessage
       id="xpack.upgradeAssistant.esDeprecations.batchReindexingDocsDescription"
-      defaultMessage="To start multiple reindexing tasks in a single request, use the Kibana {docsLink}."
+      defaultMessage="To start upgrading multiple reindexing tasks in a single request, use the Kibana {docsLink} for indices. The {docsLink} does not support data streams."
```

Comment:
Nit: `To start upgrading multiple reindexing tasks`, this sounds like we are upgrading reindexing tasks. Perhaps something like:

```suggestion
      defaultMessage="To start upgrading multiple indices in a single request, use the Kibana {docsLink}. Note: this API does not support data streams."
```

Reply:
+1 on the suggestion from @jloleysens, LGTM like that